.gitignore

```diff
@@ -7,3 +7,4 @@
 *.out
 build/
 linux_build.sh
+shard.lock
```

Comment:
As `scry` is an app first, not a library, I think it would make sense to have the `shard.lock` under source control, so we can have predictable dependencies version (in this case, for `lsp`)